src/components/MessageBubble/MessageBubble.css

```diff
@@ -0,0 +1,96 @@
+.message-container {
+    display: flex;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    font-family: 'Inter';
+    font-style: normal;
+    font-size: 16px;
+    align-items: center;
+    font-weight: 400;
+    line-height: 24px;
+    /* identical to box height, or 150% */
+    letter-spacing: 0.01em;
+}
+
+.bubble-from-right {
+    justify-content: flex-end;
+}
+
+.bubble-from-left {
+    justify-content: flex-start;
+}
+
+.bubble {
+    background-color: #FFFFFF;
+    border-radius: 25px;
+    display: block;
+    padding: 10px 21px;
+    position: relative;
+    color: black;
+    word-wrap: break-word;
+}
+
+@media (min-width: 0px) {
+  .bubble-img {
+    max-width: 70%;
+  }
+
+  .bubble-text {
+    max-width: 80%;
+  }
+}
+
+@media (min-width: 600px) {
+  .bubble-img {
+    max-width: 50%;
+  }
+
+  .bubble-text {
+    max-width: 60%;
+  }
+}
+
+@media (min-width: 768px) {
+  .bubble-img {
+    max-width: 40%;
+  }
+
+  .bubble-text {
+    max-width: 50%;
+  }
+}
+
+@media (min-width: 992px) {
+  .bubble-img {
+    max-width: 30%;
+  }
+
+  .bubble-text {
+    max-width: 40%;
+  }
+}
```

Comment:
```suggestion
@media (max-width: 0px) {
  .bubble-img {
    max-width: 70%;
  }

  .bubble-text {
    max-width: 80%;
  }
}

@media (max-width: 600px) {
  .bubble-img {
    max-width: 50%;
  }

  .bubble-text {
    max-width: 60%;
  }
}

@media (max-width: 768px) {
  .bubble-img {
    max-width: 40%;
  }

  .bubble-text {
    max-width: 50%;
  }
}

@media (max-width: 992px) {
  .bubble-img {
    max-width: 30%;
  }

  .bubble-text {
    max-width: 40%;
  }
}
```

Reply:
Should be 'max-width' for responsive resizing.

Reply:
@colaMeowSY to switch to max-width I think you need to reverse the order:
```css
@media (max-width: 992px) {
  .bubble-img {
    max-width: 30%;
  }
  .bubble-text {
    max-width: 40%;
  }
}

@media (max-width: 768px) {
  .bubble-img {
    max-width: 40%;
  }
  .bubble-text {
    max-width: 50%;
  }
}

@media (max-width: 600px) {
  .bubble-img {
    max-width: 50%;
  }
  .bubble-text {
    max-width: 60%;
  }
}
```
also the last one max-width: 0px doesn't work.
```css
@media (max-width: 0px) {
  .bubble-img {
    max-width: 70%;
  }
  .bubble-text {
    max-width: 80%;
  }
}
```

u might need to change the order for it to work the same as min-width. (would be nice if you revert back to min-width, it would be much easier)